contracts/pangolin-roll/PangolinRoll.sol

```diff
@@ -0,0 +1,175 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.6.0;
+
+import "../pangolin-core/interfaces/IPangolinPair.sol";
+import "../pangolin-core/interfaces/IPangolinFactory.sol";
+import "../pangolin-periphery/interfaces/IERC20.sol";
+import "../pangolin-periphery/interfaces/IPangolinRouter.sol";
+import "../pangolin-periphery/libraries/PangolinLibrary.sol";
+import "./libraries/SafeERC20.sol";
+import "./libraries/Ownable.sol";
+
+// PangolinRoll helps your migrate your existing Sushiswap LP tokens to Pangolin LP ones
+contract PangolinRoll is Ownable {
+    using SafeERC20 for IERC20;
+
+    IPangolinRouter public oldRouter;
+    IPangolinRouter public router;
+    IERC20 public pngToken = IERC20(0x60781C2586D68229fde47564546784ab3fACA982); // png token address
```

Comment:
let's have this passed in as a constructor argument to allow this on other networks and for ease of testing. these variables can be `immutable` to accommodate this